### PR TITLE
DPAD-1068 :: Move Close button down on Mobile Ad

### PR DESCRIPTION
**Pull request by p-forfa**
* The close button needs to be moved down, below the Learn More overlay, whenever an ad plays
* Added in a react hook to check whether the player is in an ad or not
* The close button is moved down by a static amount of 40px. Grabbing the Learn More button element is slightly difficult, given that the element exists inside of an iFrame that's sourced from a different domain

### CODE REVIEW

File: src/utils/useAdBreak.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { PlayerContext } from 'jwplayer/players/shared/PlayerContext';
+import JWEvents from 'jwplayer/players/shared/JWEvents';
+
+export default function useAdBreak(): boolean {
+	const [adBreak, setAdBreak] = useState(false);
+	const { player } = useContext(PlayerContext);
+
+	useEffect(() => {
+		player?.on(JWEvents.AD_BREAK_START, () => {
+			console.log('AD_BREAK_START');
```
Comment (p-forfa) [inline src/utils/useAdBreak.ts:11]: This ad_break_start event does not seem to fire whenever the ad plays

File: src/utils/useAdBreak.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			console.log('AD_LOADED');
+			setAdBreak(true);
+		});
+		player?.on(JWEvents.AD_PLAY, () => {
+			console.log('AD_PLAY');
+			setAdBreak(true);
+		});
+
+		player?.on(JWEvents.AD_BREAK_END, () => {
+			console.log('AD_BREAK_END');
```
Comment (p-forfa) [inline src/utils/useAdBreak.ts:28]: Same thing with this event.

File: src/utils/useAdBreak.ts (hunk 1 above)
Comment (gundams-and-dragons) [reply src/utils/useAdBreak.ts:11]: We may not be triggering this with the way we're loading ads.

Comment (p-forfa) [reply src/utils/useAdBreak.ts:11]: Got it. Maybe that'll get fixed if we utilize the built in JW API ad methods sometime in the future.

**Review by AlburIvan — APPROVED** [review]
lgtm

**Pull request merged**